pkg/crc/segment/segment.go

```diff
@@ -86,6 +94,15 @@ func (c *Client) Upload(action string, duration time.Duration, err error) error
 	})
 }
 
+func addConfigTraits(c *crcConfig.Config, in analytics.Traits) analytics.Traits {
+	return in.
+		Set(config.NetworkMode, c.Get(config.NetworkMode).AsString()).
+		Set(config.HTTPProxy, c.Get(config.HTTPProxy).AsString() != "").
```

Comment:
We are not allowed to capture these as they can contain credentials for HTTP proxy authentication, or even the naming, like `int-proxy.[companyname]`, etc.

Reply:
I understand `c.Get(config.HTTPProxy).AsString() != ""` is treated as a bool... but make this more obvious from proceeding code. Values should never leak into here by someone removing this part.

Reply:
extracted, it is more obvious now.